### ISSUE TITLE
WINPYWORKDIR environment variable - Portable VSC with PlatformIO and Python 3.9.1 for Windows 7 (64bit)

### ISSUE THREAD

**Issue by kometa2018** [Jason2866/Portable_VSC_PlatformIO#5]
I downloaded from github [https://github.com/Jason2866/Portable_VSC_PlatformIO/releases/tag/1.6](url) "Portable VSC with PlatformIO and Python 3.9.1 for Windows (64bit)" - 
unpacked, run VS Code.exe - nothing happens.
It looks like the WINPYWORKDIR environment variable is not created in bat files.
When (where) should the WINPYWORKDIR environment variable be set?

**Comment by Jason2866** [comment]
Just tried on a plain new Windows VM. Does work.
Have you extracted the whole zip file?

**Comment by kometa2018** [comment]
unzipped with winrar

vm - means virtual machine?

I unzipped and ran on windows 7 64bit

**Comment by Jason2866** [comment]
yes, VM virtual Machine. Windows 11 Pro. Unzipped with 7zip. The bundle was built on my old Laptop running Windows10 pro. Don't use Windows anymore. Got annoyed from the frequent updates. Was a Windows user since v3.11
Never tested the bundle with Windows 7...

**Comment by kometa2018** [comment]
I have a laptop with windows 10. On it just unzip and should work?

I'm going to try now...

**Comment by Jason2866** [comment]
yeah, just unzip. Recommend 7zip since it is waaaay faster than the onboard zip.

**Comment by kometa2018** [comment]
on windows 10 - opened the window "Visual Studio Code 1.73.1"
It's good, I'll keep trying.
(my working machine is with windows 7 - so I would like it to work on windows 7 too)
Thanks for the help!

**Comment by Yoti** [comment]
Python 3.9+ doesn't work on Win 7. Latest Python 3 with Win 7 support is Python 3.8.

**Comment by Jason2866** [comment]
@Yoti Thx, for the info. Windows 7 is EoL. Won't can't fix. Don't have a build env with Win7. Stopped using Windows. Now on Mac. The Windows VM using to build is Win11 ARM pro.
